src/index.js

```diff
@@ -1,7 +1,6 @@
 const babelPluginMagentoLayout = require('./babel-plugin-magento-layout');
-const MagentoRootComponentsPlugin = require('./MagentoRootComponentsPlugin');
 
 module.exports = {
     babelPluginMagentoLayout,
-    MagentoRootComponentsPlugin
+    Webpack: require('./webpack')
```

Comment:
`Webpack` isn't a constructor, so it would make sense to follow JS conventions and use lowercase

Reply:
True. Also keep in mind that this object will be imported by `webpack.config.js`, where `webpack` will already be present as the logical import from `webpack` itself.

Maybe just call this `buildpack`, after the module?

Reply:
@DrewML I don't think it's established JS conventions that PascalCase only applies to constructors. As a quick sample, [Airbnb](https://github.com/airbnb/javascript#naming--PascalCase-singleton) applies PascalCase much more broadly.

@jimbo Very true that we have to find ways to make this not collide with common stuff in `webpack.config.js`. I can't rename it `buildpack`, because this grouping is specifically for objects that work in the Webpack lifecycle, be they configurators or plugins or loaders.

I think I'm going to rename it to `WebpackTools`. I'm also going to say we should use PascalCase for all singleton objects, not just constructors. I understand that this decision is arbitrary, but I'm considering the many PHP developers who will be reading lots of JS for the first time. The distinction between a constructor and a plain object might not be immediately clear, and the capitalization rules should appear typographically consistent and familiar.